### ISSUE TITLE
Change to bootstrap 5 template

### ISSUE THREAD

**Issue by diorz38** [riteshsingh1/livewire-crud#4]
Default generated balde was tailwind, how to change to bootstrap 5

**Comment by diorz38** [comment]
never mind, i found it in config, forgot to publish it

**Comment by riteshsingh1** [comment]
Noted, Currently this support of bootstrap 4  only, 

Bootstrap 5 will be released soon.

**Comment by ehsantalkhouncheh** [comment]
Hi If we want just tailwind should we do anything special i mean default them is tailwind right?

**Comment by damodar-bhattarai** [comment]
> Hi If we want just tailwind should we do anything special i mean default them is tailwind right?

Yes. Default one is tailwind. No need to do anything if you want tailwind.
For bootstrap 4 , publish config file and change value of template variable to "bootstrap"
